docs/about/history.rst

```diff
@@ -40,6 +41,11 @@ Changed
 
    -  Warn users edits in prefab mode will not be reflected in scene view until prefab is saved.
 
+Changed
+^^^^^^^
+.. bullet_list::
+
+   -  Added ability to blend waves (effectively an override) instead of only having additive waves. Disable "Additive" on the *ShapeFFT* or *ShapeGerstner* when using with a *MeshRenderer* or *Spline* to use. Useful for preventing rivers from receiving ocean waves.
```

Comment:
wording out of date now that you have the enum, and i'd probably say *Added ability to alpha blend waves* as additive is a type of blend. might just be a graphics-tainted mind reading it that way though :)

Reply:
Good point. Done.